docs/developer-guide/cicd/README.md

```diff
@@ -4,4 +4,19 @@ order: 13
 
 # Create a CI/CD pipeline for Kestra
 
+## Introduction
+
+In Kestra, the number of flows you developed can grow quickly, and maintaining them can be a challenge.
+CI/CD pipelines are a great way to automate the validation and the deployment of your flows.
+This section will guide you through the steps to create a CI/CD pipeline.
+
+## Why a CI/CD pipeline?
+
+CI/CD pipelines provide an easy and reliable way to ensure that your flows are always deployed as soon they're done.
```

Comment:
As soon as they are done with? 👀

Reply:
> are always deployed as soon they're in your source control.

That's the point of a continuous integration right ? You commit, it trigger a workflow

Reply:
`they are in your source control.`

Yeah, that makes sense. I wasn't sure if there was more to it. Thanks :)